### DISCUSSION TITLE
Change the Homepage Image

### DISCUSSION THREAD

**Discussion by lintinglai** [saleor/saleor#7805] [Q&A]
Hello Teams,

Just a small question, how to change the homepage image? I could not find it on dashboard. If I need to enter PostgreSQL to change the "backgroundImage.url"? Or other ways? Thanks.

/../saleor-platform/saleor-storefront/src/views/Home/Page.tsx

`<div
        className="home-page__hero"
        style={
          backgroundImage
            ? { backgroundImage: `url(${backgroundImage.url})` }
            : null
        }
      >
        <div className="home-page__hero-text">
          <div>`

**Comment by karolkielecki** [comment]
@lintinglai Hi there! You can find more details on how to modify your storefront, including editing images, [here](https://spectrum.chat/saleor/saleor-storefront/modifying-the-storefront~c1955dbf-a421-4fb6-b99e-937dd2642b23).

**Reply by lintinglai** [reply]
Thanks, Karol! Update quickly, I could query those data through GraphQL. I think it's necessary to  understand React, GraphQL and may need Apollo tool.

![image](https://user-images.githubusercontent.com/87848789/129313221-bdef06cb-4833-465b-95b4-7f6c2cad3f58.png)

**Reply by karolkielecki** [reply]
@lintinglai Yes, it demands to be familiar with React and GraphQL.

**Comment by Megidd** [comment]
At least for version `2.11`, you don't need any coding to change the home page images. The home page images are just the background images for *collections* and *categories* which you can easily modify by dashboard:

![Screenshot_20210814_182433](https://user-images.githubusercontent.com/17475482/129448725-55231271-37d4-44d0-a242-fa6e4d4bbcfe.png)

On the dashboard, a *collection* background image can be set here:

![Screenshot_20210814_183120](https://user-images.githubusercontent.com/17475482/129448835-6ff20249-7bd5-4746-ad14-9e062154ffd6.png)

**Reply by lintinglai** [reply]
@Megidd Thank you so much! I just saw the collection item on dashboard. It's really helpful, thank you!
